components/brave_wallet_ui/components/desktop/wallet-sub-view-layout/index.tsx

```diff
@@ -2,14 +2,15 @@ import * as React from 'react'
 import { StyledWrapper } from './style'
 
 export interface Props {
+  isSwap?: boolean
   children?: React.ReactNode
 }
 
 export default class WalletSubViewLayout extends React.PureComponent<Props, {}> {
   render () {
-    const { children } = this.props
+    const { children, isSwap } = this.props
     return (
-      <StyledWrapper>
```

Comment:
We should call this prop something generic instead of `isSwap` - ideally something that works for all full-width screens.

Reply:
Renamed one to `maintainWidth` and the other to `noPadding`